Example/Tests/TestUtils.swift

```diff
@@ -55,12 +55,12 @@ struct TestUtils {
         let mockTokenResponse = OIDTokenResponse(
             request: mockTokenRequest,
             parameters: [
-                "access_token": mockAccessToken as NSCopying & NSObjectProtocol,
-                "expires_in": expiresIn as NSCopying & NSObjectProtocol,
-                "token_type": "Bearer" as NSCopying & NSObjectProtocol,
-                "id_token": mockIdToken as NSCopying & NSObjectProtocol,
+                 "access_token": mockAccessToken as NSCopying & NSObjectProtocol,
```

Comment:
nit: Did anything change in this file at all? I only see indentation changing. If so, can we revert this?

Reply:
This is mostly a style thing, and keeps us consistent with other objects within this file. Happy to revert back the style changes if needed.

Reply:
@jmelberg-okta, I am fine with it if it is an intended change. Thx.